ffi/koptcontext.lua

```diff
@@ -100,9 +97,6 @@ function KOPTContext_mt.__index:getWordBoxes(bmp, x, y, w, h, box_type)
     if boxa == nil or nai == nil then return end
 
     --get number of lines in this area
-    leptonica.numaGetMax(nai, max_val, last_index)
-    nr_line = max_val[0]
-    --get number of lines in this area
```

Comment:
I suppose nr_line was not used anywhere. Hoping leptonica.numaGetMax() don't do any stuff on the side

Reply:
I could also add luacheck ignores to 'em but selecting words didn't seem to act any differently for any of this.

Reply:
Ok then (I rarely read pdf, so you'll probably notice sooner than me if it has rare side effects)